sys-fs/zfs/zfs-9999.ebuild

```diff
@@ -1,6 +1,12 @@
 # Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+#!/bin/bash
+
+# Adding Portage related postinst warnings
+ewarn "This package may require a rebuild after upgrading Portage."
```

Comment:
This doesn't belong in global scope but I also have no idea what this is referring to. You don't need to rebuild ZFS after upgrading Portage.

What exactly is this addressing?